nerfstudio/data/dataparsers/nuscenes_dataparser.py

```diff
@@ -200,10 +200,10 @@ def _generate_dataparser_outputs(self, split="train"):
         )
 
         cameras = Cameras(
-            fx=intrinsics[:, 0, 0],
-            fy=intrinsics[:, 1, 1],
-            cx=intrinsics[:, 0, 2],
-            cy=intrinsics[:, 1, 2],
+            fx=intrinsics[:, 0, 0].detach().clone(),
+            fy=intrinsics[:, 1, 1].detach().clone(),
+            cx=intrinsics[:, 0, 2].detach().clone(),
+            cy=intrinsics[:, 1, 2].detach().clone(),
             height=900,
             width=1600,
             camera_to_worlds=poses[:, :3, :4],
```

Comment:
should we also do this on the transform matrices?

Reply:
No, this is not necessary, because the data in `poses` is not shared across multiple parameters as with fx, fy, cx, cy of the tensor `intrinsics`.

With my fix applied, `ns-train nerfacto nuscenes-data --data ... --data_dir ...` runs as expected.